source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,95 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``*Collection.FindOne()`` method. 
+
+..
+   method
+
+The ``FindOne()`` method takes a query document that specifies the
+subset of your collection that you would like to retrieve a document from. If you
+pass the ``FindOne()`` method an empty document, MongoDB will consider your
+entire collection when retrieving your document. For more information on documents
+in the MongoDB Go driver, see our
+:doc:`guide on data formats </fundamentals/data-formats>`.
+
+You must pass the ``FindOne()`` method a ``context.Context`` interface type
+value. For more information on the ``context.Context`` interface and the
+``context`` package, see our :doc:`guide on context </fundamentals/context>`.
+
+..
+   options
+
+You can specify options that modify the behavior of your ``FindOne()`` method.
+One example of an option is **sort**, which orders the subset of your collection
+that matches your query before the first document is retrieved. Another example
+of an option is a **projection**, which specifies or restricts the fields in
+your returned document. You can specify additional options by passing the
+``FindOne()`` method a pointer to a ``FindOneOptions`` struct. You
+can configure a ``FindOneOptions`` struct by using setter methods such as the
+``*FindOneOptions.SetProjection()`` method.
+
+..
+   return
+
+If the query filter you specified in your ``FindOne()`` method call matches one
+or more documents in the collection, it returns the first matched result. The
+return type ``SingleResult`` represents a single document returned from a query
+to a MongoDB instance. You can call the ``*SingleResult.Decode()`` method
+to unmarshall a ``SingleResult`` value into a variable. Pass the ``Decode()`` 
```

Comment:
```suggestion
to unmarshal a ``SingleResult`` value into a variable. Pass the ``Decode()`` 
```

Reply:
Thanks for the note. Going forward, I will try to remember that "[Marshal](https://golang.org/pkg/encoding/json/#Marshal)" is spelled with one "L" in the go ecosystem. 

Committing suggested change.